README.md

```diff
@@ -78,6 +81,17 @@ The finished web app is found in the `docs/` folder (this is so that you can eas
 
 You can test the template app at <https://emilk.github.io/eframe_template/>.
 
+### Using Trunk
+
+* Trunk uses `index.html` from the project folder and copies some files from `docs` folder into a new temporary folder called `dist`. 
+* `trunk serve` will build the app and host it on `http://127.0.0.1:8080`. no scripts or the need to edit any files. 
+* because of `docs/sw.js` script, the webpage will try to cache the app, so try force refreshing `Ctrl + F5` to force the webpage to pull in the new updated website.
+
+## Github Pages
+* `.github/workflows/pages.yml` will run on every commit and deploy the egui app to `gh-pages` branch of your repo.
+* you need to go to Respository -> Settings -> Pages -> Source -> set to `gh-pages` branch and `/` (root). this is a one time setup, otherwise, githug pages deployment won't work. and this will only be visible after the `pages.yml` workflow has run atleast once (initial commit is fine).
```

Comment:
Does this mean the built wasm binary is commited to git on each CI step? It seems like this will create a huge git repository pretty fast, no?